lib/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -750,14 +751,102 @@ public static function get_style_variations() {
 		foreach ( $variation_files as $path => $file ) {
 			$decoded_file = wp_json_file_decode( $path, array( 'associative' => true ) );
 			if ( is_array( $decoded_file ) ) {
-				$translated = static::translate( $decoded_file, wp_get_theme()->get( 'TextDomain' ) );
-				$variation  = ( new WP_Theme_JSON_Gutenberg( $translated ) )->get_raw_data();
+				$translated           = static::translate( $decoded_file, wp_get_theme()->get( 'TextDomain' ) );
+				$variation_theme_json = new WP_Theme_JSON_Gutenberg( $translated );
+				$resolved_theme_uris  = WP_Theme_JSON_Resolver_Gutenberg::get_resolved_theme_uris( $variation_theme_json );
+				$variation            = $variation_theme_json->get_raw_data();
 				if ( empty( $variation['title'] ) ) {
 					$variation['title'] = basename( $path, '.json' );
 				}
+				/*
+				 * @TODO this needs to be added to the WP REST API schema.
+				 * E.g., $links['https://api.w.org/theme-file-uris']. ??
+				 */
+				/*
+				 * @TODO I'm not sure if we can/should add `_links` to collection items.
+				 * It should be part of the response object, but given that
+				 * WP_REST_Global_Styles_Controller_Gutenberg::get_theme_items
+				 * returns a collection ([{}]) and not a response object ({}) it's not possible.
+				 */
+				if ( ! empty( $resolved_theme_uris ) ) {
+					$variation['_links']['wp:theme-file-uris'] = $resolved_theme_uris;
```

Comment:
Looks like we should add this stuff to the header:

https://developer.wordpress.org/rest-api/using-the-rest-api/linking-and-embedding/